### ISSUE TITLE
Alignment traceback error

### ISSUE THREAD

**Issue by tobiasrausch** [maickrau/GraphAligner#76]
Hi,

When I generate exact 5,000bp reads from the chm13 reference and map them back to a pan-genome graph that includes chm13 about ~1% of all reads have a CIGAR of cg:Z:4999=63D1X instead of the expected cg:Z:5000=

Here is a small example of 3 reads reproducing the 4999=63D1X CIGAR string

```
wget https://zenodo.org/record/6499594/files/chm13-90c.r518.gfa.gz
gfatools gfa2fa -s -0 chm13-90c.r518.gfa.gz > chm13.fa
samtools faidx chm13.fa chr1:17039501-17044500 chr1:17047501-17052500 chr1:17078001-17083000 > reads.fa
gunzip chm13-90c.r518.gfa.gz 
GraphAligner -t 24 -b 35 -g chm13-90c.r518.gfa -f reads.fa -a align.gaf -x vg
```

Best, Tobias

**Comment by maickrau** [comment]
This should now be fixed in [83f3daf](https://github.com/maickrau/GraphAligner/commit/83f3daf2561aa3536bbd91b3e1d80b34b5d8e822)

**Comment by tobiasrausch** [comment]
Thanks, looks good now.
